server/i18n/en.json

```diff
@@ -5082,6 +5082,10 @@
     "id": "app.import.bulk_import.unsupported_version.error",
     "translation": "Incorrect or missing version in the data import file. Make sure version is the first object in your import file and try again."
   },
+  {
+    "id": "app.import.custom_status.error",
+    "translation": "Unable to marshal custom status"
```

Comment:
Looking at the code this isn't necessarily about marshaling. Can be a more generic "Unable to set customer status".

Reply:
Yeah I mean the failure will eventually come because of failure to marshal, but I agree we should have a high level message instead. The internal error is anyways going to be logged.